easybuild/easyconfigs/n/NanoPlot/NanoPlot-1.42.0-foss-2022a.eb

```diff
@@ -0,0 +1,44 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'NanoPlot'
+version = '1.42.0'
+
+homepage = 'https://github.com/wdecoster/NanoPlot'
+description = "Plotting suite for long read sequencing data and alignments"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['0f8fd2cffd33a346b3306716058c6cb4091c931e8ab502f10b17a28749e8b6d9']
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('Biopython', '1.79'),
+    ('Pysam', '0.19.1'),
+    ('nanomath', '1.3.0'),
+    ('nanoget', '1.19.1'),
+    ('Seaborn', '0.12.1'),
+    ('plotly.py', '5.12.0'),
+    ('pauvre', '0.2.3'),
```

Comment:
I'm wondering if this still needs Seaborn and pauvre: the setup.py of this version (https://github.com/wdecoster/NanoPlot/blob/v1.42.0/setup.py#L34) doesn't list them anymore, while the one for 1.3.0 did (https://github.com/wdecoster/NanoPlot/blob/1.33.0/setup.py#L43). There's a specific commit (https://github.com/wdecoster/NanoPlot/commit/f977a26ff9f3c6e76d80e2c10f36c95a4d603457) where they're dropping them, and one where the `pauvre_plot` function gets commented (https://github.com/wdecoster/NanoPlot/commit/17bde314ca69776246d9ecab8639b9dee9832730).

Reply:
Absolutely right. https://github.com/easybuilders/easybuild-easyconfigs/pull/20338/commits/e81fefba03d96b544b1b24eefb543dba4c654ae4 shall address that.